mypyc/ops.py

```diff
@@ -1566,6 +1567,18 @@ def subclasses(self) -> Set['ClassIR']:
                 result.update(child.subclasses())
         return result
 
+    def concrete_subclasses(self) -> List['ClassIR']:
```

Comment:
This is a bit inconsistent with `subclasses()`, since the latter doesn't include the current type. I'd leave it out here, and the letter the caller add it.

Reply:
OK.